### PR TITLE
docs: Add recent contributors

### PR DESCRIPTION
**Pull request by arielvalentin**
This change adds contributors to codeowners for contrib packages:

@chrisholmes
@johnnyshields
@indrekj
@muripic
@NathanielRN
@rmosolgo
@swalkinshaw

Thank you all for your contributions!

In order for this change to work as expected we need for the members listed above to join the OTel GitHub Org https://github.com/orgs/open-telemetry/people

Open an issue here and add my name as a sponsor:

https://github.com/open-telemetry/community/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E

### CODE REVIEW

**Comment by johnnyshields** [comment]
@arielvalentin thanks! How do I join the org?

**Comment by arielvalentin** [comment]
> @arielvalentin thanks! How do I join the org?

Please open an issue here:

https://github.com/open-telemetry/community/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E

You will need another person in the community to sponsor you (in addition to me).

**Comment by muripic** [comment]
Done ✅ 
Any volunteers for a second sponsor? 😇

**Comment by arielvalentin** [comment]
@robbkidd @mwear, as I suspected the codeowners linter wants for members to have write access. Being a member of the `open-telemetry` org is not enough:

image

File: CODEOWNERS [hunk 1]
```diff
@@ -13,3 +13,21 @@
...
+instrumentation/grape/ @muripic @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd
+
+instrumentation/graphql/ @swalkinshaw @rmosolgo @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd
+
+instrumentation/mongo/ @johnnyshields @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd
+
+instrumentation/racecar/ @chrisholmes @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd
+
+instrumentation/rdkafka/ @andyfleming @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd
+
```
Comment (arielvalentin) [inline CODEOWNERS:30]:
```suggestion
```

**Comment by robertlaurin** [comment]
> as I suspected the codeowners linter wants for members to have write access

What does write access enable?

**Comment by arielvalentin** [comment]
@robertlaurin https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization

**Comment by arielvalentin** [comment]
I think the contentious one is "Merge a PR".

**Comment by mwear** [comment]
> Done ✅ [open-telemetry/community#1624](https://github.com/open-telemetry/community/issues/1624) Any volunteers for a second sponsor? 😇

@muripic, I'll sponsor.

**Comment by muripic** [comment]
> > Done ✅ [open-telemetry/community#1624](https://github.com/open-telemetry/community/issues/1624) Any volunteers for a second sponsor? 😇
> 
> @muripic, I'll sponsor.

Just added you, thank you! 🙇‍♀️

**Review by ericmustin — APPROVED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -13,3 +13,19 @@
 #
 
 * @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd
```
Comment (mwear) [inline CODEOWNERS:15]: Can this list be shortened to `@open-telemetry/ruby-contrib-approvers`? If so, we can probably do that on every line.

Comment (arielvalentin) [reply CODEOWNERS:15]: IIRC That was by request from @fbogsany and @robertlaurin when I first set up the original codeowners file.

If there are no objections from them I will change it.

Comment (mwear) [reply CODEOWNERS:15]: I'm not sure if there is a reason to prefer one format over the other, but this is how collector-contrib is setup: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS. I see two benefits. When we add or remove approvers we won't have to make changes to CODEOWNERS. Also, it makes it easier to see who the supplemental code owners are.

Comment (arielvalentin) [reply CODEOWNERS:15]: > I'm not sure if there is a reason to prefer one format over the other, but this is how collector-contrib is setup: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS. I see two benefits. When we add or remove approvers we won't have to make changes to CODEOWNERS. Also, it makes it easier to see who the supplemental code owners are.

I agree with @mwear on this FWIW

**Review by ahayworth — APPROVED** [review]

**Pull request merged**